test/twine_test.coffee

```diff
@@ -6,9 +6,9 @@ suite "Twine", ->
     Twine.reset(context, rootNode).bind().refreshImmediately()
     rootNode.children[0]
 
-  triggerEvent = (node, eventName) ->
-    event = document.createEvent("HTMLEvents")
-    event.initEvent eventName, false, true
+  triggerEvent = (node, eventName, data = {}) ->
+    event = document.createEvent('CustomEvent')
+    event.initCustomEvent eventName, false, true, data
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/initCustomEvent cites deprecation warning

May be related to your data failing to pass thru in the test below?